include/host-build.mk

```diff
@@ -52,6 +52,7 @@ HOST_CONFIGURE_VARS = \
 	CXX="$(HOSTCXX)" \
 	CPPFLAGS="$(HOST_CPPFLAGS)" \
 	LDFLAGS="$(HOST_LDFLAGS)" \
+	CXXFLAGS="$(HOST_CXXFLAGS)" \
```

Comment:
Please put this after the CPPFLAGS, same for the other change

Reply:
done